docs/reference/upgrade.asciidoc

```diff
@@ -9,14 +9,20 @@ process so upgrading does not interrupt service. Rolling upgrades are supported:
 * Between minor versions
 * From 5.6 to 6.8
 * From 6.8 to {version}
+* Between {minor-version} versions
```

Comment:
I wonder if "between" might be read as implying you can change version in both directions within the 7.6.x series. How about "from any version since `{major-version}.0` to {version}" and drop the "Between minor versions" line?

Reply:
Good suggestion. Updated with 4d2cc9029e4.